lib/datadog/tracing/contrib/propagation/sql_comment/comment.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# typed: false
+
+require 'erb'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Propagation
+        module SqlComment
+          # To be prepended to a sql statement.
+          class Comment
+            def initialize(hash)
+              @hash = hash
+            end
+
+            def to_s
+              @string ||= begin
+                ret = String.new
+
+                @hash.each do |key, value|
+                  next if value.nil?
+
+                  value = ERB::Util.url_encode(value) # url encode
+                  value.tr!("'", "\'") # escaping single quote
```

Comment:
Given this:
```ruby
ERB::Util.url_encode("'")
# => "%27"
```

I don't think we can have `'` in `value` anymore at this point. I think we can remove this `tr!` call.

Also, the `tr!` arguments seem incorrect, given they are the same: 
```ruby
"'" == "\'"
# => true
```